### PR TITLE
added rdma_rename to Ubuntu image

### PR DESCRIPTION
**Pull request by vgamayunov**

### CODE REVIEW

**Comment by vgamayunov** [comment]
found another issue with IB on Ubuntu image - will add to this PR once tested

**Comment by vgamayunov** [comment]
adding FORCE_MODE=yes to openib.conf fixes the issue. The adapter names are coming up as expected. Ready to merge

File: packer/scripts/ubuntu/infiniband.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+echo " ********************************************************************************** "
+echo " *                                                                                * "
+echo " *     INFINIBAND CONFIGURAION                                                    * "
+echo " *                                                                                * "
+echo " ********************************************************************************** "
+set -ex
```
Comment (hmeiland) [inline packer/scripts/ubuntu/infiniband.sh:7]: is this script valid for ubuntu18 & 20 & 22? if not let's make forks

**Review by hmeiland — APPROVED** [review]

File: packer/scripts/ubuntu/infiniband.sh (hunk 1 above)
Comment (hmeiland) [reply packer/scripts/ubuntu/infiniband.sh:7]: also propose to add this to the PG repo for ubuntu-hpc

Comment (vgamayunov) [reply packer/scripts/ubuntu/infiniband.sh:7]: agree with both points. at present we only have 18.04 image and will need to verify all scripts for 20 and 22.

**Pull request merged**